packages/client/components/AnalyticsPage.tsx

```diff
@@ -200,8 +212,23 @@ const AnalyticsPage = () => {
         {integrations: {'Google Analytics': {clientId: await getAnonymousId()}}}
       )
       ReactGA.send({hitType: 'pageview', content_group: getContentGroup(pathname)})
+      amplitude.track(
```

Comment:
-1 This effect has an early return on l191 so this will not be reached of GA or Segment aren't loaded, although this should be independent. Also if any of the track calls before would throw, we'd never reach this point. I suggest splitting this up so each of the track functions could work independently.